src/classes/UTIL_Describe_TEST.cls

```diff
@@ -132,4 +132,40 @@ public with sharing class UTIL_Describe_TEST {
             System.assert(false, 'Always throw an exception when NULL is passed as a parameter');
         } catch (System.NullPointerException npe) { }
     }
+
+    //Test for getCustomAdminAccRecTypeId if the value of Administrative Account Record Type is empty
+    @isTest
+    private static void getCustomAdminAccRecTypeIdEmptyTest() {                      
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Administrative_Account_Record_Type__c = null));
+        
+        Test.startTest();
+            Id testCustomAdminAccRecTypeId = Util_Describe.getCustomAdminAccRecTypeId();
+        Test.stopTest();
+
+        System.assertEquals(Util_Describe.getAdminAccRecTypeID(), testCustomAdminAccRecTypeId);
+    }
+
+    //Test for getCustomAdminAccRecTypeId if the value of Administrative Account Record Type is not empty but the record type does not exist
+    @isTest
+    private static void getCustomAdminAccRecTypeIdNotEmptyNotExistTest() {    
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Administrative_Account_Record_Type__c = '000000000000000000'));
+        
+        Test.startTest();
+            Id testCustomAdminAccRecTypeId = Util_Describe.getCustomAdminAccRecTypeId();
+        Test.stopTest();
+
+        System.assertEquals(Util_Describe.getAdminAccRecTypeID(), testCustomAdminAccRecTypeId);
+    }
+
+    //Test for getCustomAdminAccRecTypeId if the value of Administrative Account Record Type is not empty and the record type exists
+    @isTest
+    private static void getCustomAdminAccRecTypeIdNotEmptyExistsTest() {
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Administrative_Account_Record_Type__c = Util_Describe_API.getRecTypesMapByDevName('Account').values()[1]));
```

Comment:
Assuming the order of the returned map of recordtypes is a really brittle practice. I know it's very prevalent in UTIL_Describe but I don't think we should propagate it here too.

Reply:
Second that! The idea here is making sure the admin account record type is not empty and testCustomAdminAccRecTypeId get same record type id as the map.values()[1].

Reply:
Agreed on it being brittle @kevinswiggum. Once we get to the unit test update work we can use the new method in our unit tests that will always handle the same record type in a consistent format.